### ISSUE TITLE
Memory leaking

### ISSUE THREAD

**Issue by mmack** [hjacobs/kube-ops-view#166]
Hey all,

i have 6 clusters connected to kube-ops-view and after half an hour it's nearly impossible to browse anymore... I'm using google chrome or chromium... i've collected 3 memory samles, but b/c of the size i'm not going to upload them here. If someone is interested in the sample i can share them...

```  
➜  opsview samples ll
total 502M
-rw-rw-r-- 1 mmack mmack  38M Jun  8 09:43 Heap-20180608T094316.heapsnapshot
-rw-rw-r-- 1 mmack mmack 131M Jun  8 09:43 Heap-20180608T094350.heapsnapshot
-rw-rw-r-- 1 mmack mmack 333M Jun  8 09:44 Heap-20180608T094355.heapsnapshot
```

**Comment by hjacobs** [comment]
Yes, I'm aware of the browser problems, that's why I added a workaround with the `reload` UI option, see http://kubernetes-operational-view.readthedocs.io/en/latest/ui-options.html (I even mention JS memory leaks there). Hope that helps.

**Comment by morsik** [comment]
I have the same problem. After one day, tab with Kube Ops used 5,5GB on my Chrome.
